crates/ide-completion/src/item.rs

```diff
@@ -454,10 +456,17 @@ impl Builder {
         if let [import_edit] = &*self.imports_to_add {
             // snippets can have multiple imports, but normal completions only have up to one
             if let Some(original_path) = import_edit.original_path.as_ref() {
-                label = SmolStr::from(format!("{label} (use {})", original_path.display(db)));
+                label_detail = Some(SmolStr::from(format!(
```

Comment:
This is a bit hard to follow, but I don't have any suggestions :shrug:.

Reply:
@lnicola I rewrote logic to avoid reassignments and force pushed it the PR branch.

Reply:
But I don't know would @bors see my latest changes and take them into account.

Reply:
@lnicola As I understand @bors will complete this PR as it saw it on the https://github.com/rust-lang/rust-analyzer/commit/f8bf1dc15d770642b252ed6e35bcfb84ef074fda commit.
Can I open an another PR with a small fix that force pushed here?